api/argo-server/swagger.json

```diff
@@ -9,7 +9,7 @@
   "info": {
     "description": "Argo Server",
     "title": "Argo Server",
-    "version": "latest"
+    "version": "kube-client"
```

Comment:
Should this be changing with merge?

Reply:
I probably need to re-run `make codegen lint`